### PR TITLE
Formalize reference standard

### PR DESCRIPTION
**Pull request by kingthorin**
* Added a section to README.md that details the reference standard.
  * I relocated the issues/PRs note to the intro section instead of the objective section.
* After this is accepted the text can also be added to the `www` repo.
* When 4.2 or 5 is nearing completion it could also be added to the "Using the ASVS" chapter in the verbose documents.

This Pull Request relates to issue OWASP/ASVS#715

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

**Comment by kingthorin** [comment]
Note: I'm not actually an ASVS user, I don't understand why the existing identifiers all seem to be prefixed with capital V. So I haven't mentioned/included that. If you want me to add capital Vs to this content let me know.

**Comment by tghosth** [comment]
@kingthorin please take a look at my changes and confirm everything still makes sense

File: README.md [hunk 1]
```diff
@@ -19,7 +21,20 @@ The requirements were developed with the following objectives in mind:
 * Assist organizations to benchmark application security tools by the percentage of coverage of the ASVS for dynamic, interactive, and static analysis tools
 * Minimize overlapping and competing requirements from other standards, by either aligning strongly with them (NIST 800-63), or being strict supersets (OWASP Top 10 2017, PCI DSS 3.2.1), which will help reduce compliance costs, effort, and time wasted in accepting unnecessary differences as risks.
 
-**Please [log issues](https://github.com/OWASP/ASVS/issues) if you find any bugs or if you have ideas. We may subsequently ask you to [open a pull request](https://github.com/OWASP/ASVS/pulls) based on the discussion in the issue. We are also actively looking for translations of the 4.n branch.**
+### How To Reference ASVS Requirements
+
+Each requirement has an identifier in the format `<chapter>.<segment>.<requirement>` where each element is a number, ex: `1.11.3`. 
+- The `<chapter>` value corresponds to the chapter from which the requirement comes, ex: all `1.#.#` requirements are from the `Architecture` chapter.
```
Comment (kingthorin) [inline README.md:27]: I had been trying to avoid it being kinda: chapter, chapter, chapter 🙂 

But I’m good with this.

**Comment by kingthorin** [comment]
Looks good, thanks! 👍

**Comment by kingthorin** [comment]
Okay so per: https://github.com/OWASP/ASVS/issues/715#issuecomment-648216850 you'd like them all to have the leading V?

**Comment by kingthorin** [comment]
If you guys can decide that you want the Vs or not this could be knocked off and then tackled elsewhere like the www repo.

**Comment by kingthorin** [comment]
Anyone out there?

**Comment by kingthorin** [comment]
It would be really nice to get this done. If someone could provide a solid answer that'd be really handy.

**Comment by elarlang** [comment]
Seems abandoned repository :/ (https://github.com/OWASP/ASVS/issues/750#issuecomment-652985270)

**Comment by kingthorin** [comment]
This has been updated and hopefully completed (merge ready) based on discussion in: https://github.com/OWASP/ASVS/issues/812

**Comment by danielcuthbert** [comment]
Thanks for that, very quick. Give me a few days and ill action

**Comment by kingthorin** [comment]
Thanks for the merge, I'll open a PR for the www repo later this evening or over the weekend.

**Pull request merged**